JSQMessagesViewController/Controllers/JSQMessagesViewController.m

```diff
@@ -1093,8 +1093,13 @@ - (void)jsq_scrollComposerTextViewToBottomAnimated:(BOOL)animated
 
 - (void)jsq_updateCollectionViewInsets
 {
-    [self jsq_setCollectionViewInsetsTopValue:self.topLayoutGuide.length + self.topContentAdditionalInset
-                                  bottomValue:CGRectGetMaxY(self.collectionView.frame) - CGRectGetMinY(self.inputToolbar.frame)];
+    if ([UIDevice currentDevice].systemVersion.floatValue >= 11.0) {
+        [self jsq_setCollectionViewInsetsTopValue:self.collectionView.contentInset.top
```

Comment:
There seem to have a few occurrence of `self.topLayoutGuide.length + self.topContentAdditionalInset` though. Please check if those would be a problem.